### DISCUSSION TITLE
is GET /api/1/vehicles/{id}/latest_vehicle_data supported

### DISCUSSION THREAD

**Discussion by Panda88CO** [timdorr/tesla-api#643] [Q&A] [answered]
The latest endpoint file **latest_vehicle_data** is not listed.  Does anybody know if it is working - I can call it but seems to get identical data to **vehicle_data** end point.  If the car goes offline - the response becomes None

**Comment by core-hacked** [comment, marked as answer]
This endpoint was removed, but seems to function for all cars (backwards compatibility I am guessing). 
Tesla seems to use the `vehicle_data` endpoint on the newest app version + SW Version.

**Comment by SylvainGa** [comment]
Is there a way without latest_vehicle_data to get that **latest** data before the vehicle went to sleep when that the vehicle **is** asleep? Assuming no app has that data cached, ie, having to rely on a call to Tesla's system to retrieve that latest data.

Edit: I wanted to know if the 408 received when the vehicle is asleep still contained the latest data in its stream and alas, the only thing the data stream contains is:

"error" (Lang.String): "vehicle unavailable: {:error=>"vehicle unavailable:"}" (Lang.String)
"error_description" (Lang.String): "" (Lang.String)
"response" (Lang.String): null

:-(

**Reply by superfloh247** [reply]
you can get all values including the timestamp when TeslaLogger last updated this value with :5000/get

See webserver docs https://github.com/bassmaster187/TeslaLogger/blob/master/TeslaLogger/WebServer.md

**Reply by SylvainGa** [reply]
Hi, works well when it's for yourself, but it's for a (open source) watch app for Garmin watches available to anyone with a compatible watch. I was hoping to get the latest data when you launch the app instead of having to wake it up to get this data. On many watches (those without Glance support), the app only runs when activated. Glance type watch can query the system at five minutes interval so it can still miss something.
